tox.ini

```diff
@@ -3,7 +3,7 @@
 # test suite on all supported python versions. To use it, "pip install tox"
 # and then run "tox" from this directory.
 [tox]
-envlist = build_test, codegen, py{27,34,35,36,37,38,py,py3}-test, flake8, docs, pkg
+envlist = py{27,34,35,36,37,38,py,py3}-test
```

Comment:
What was the reason for this change? The old default of do-everything-except-run-system-tests was convenient.

Reply:
The way tox work is that it will run each environment in the envlist, even if previous environments fail.

It also has the ability to expand expressions so something like `py{27,36}-test` becomes `py27-test` and `py36-test`.

I find it much easier to have command that I use to build that explicitly calls each environment so that it can stop if their is a failure. Doing this though can't take advantage of the expansion.